SobokSobok/SobokSobok/Presentation/Medicine/AddMedicineSheet/AddMedicineTableViewCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AddMedicineTableViewCell.swift
+//  SobokSobok
+//
+//  Created by 김승찬 on 2022/01/09.
+//
+
+import UIKit
+
+class AddMedicineTableViewCell: UITableViewCell {
+
+    @IBOutlet var medicineImageView: UIImageView!
+    @IBOutlet var addMedicineLabel: UILabel!
```

Comment:
```suggestion
    @IBOutlet var medicineTextLabel: UILabel!
```
네이밍이 요렇게 바뀌는게 더 자연스러울 것 같은데 어떻게 생각하시나요?
(그냥 진짜 의견 제시니까 가볍게 넘어가도 됩니다!)

Reply:
네이밍에 대해 고민해봐야겠스빈당..